src/txEncoders.ts

```diff
@@ -91,14 +91,23 @@ export const encodeTxBody = (txBody: TransactionBody) => new Map(([
     [9, txBody.mint && encodeMultiasset(txBody.mint)],
 ]).filter(([_, value]) => value !== undefined) as [number, unknown][])
 
-export const encodeSignedTx = (signedTx: SignedTransaction) => [
-    encodeTxBody(signedTx.body),
-    signedTx.witnessSet,
-    signedTx.auxiliaryData,
+export const encodeTx = (tx: Transaction) => [
+    encodeTxBody(tx.body),
+    tx.witnessSet,
+    tx.auxiliaryData,
 ]
 
-export const encodeRawTx = (rawTx: RawTransaction) => [
-    encodeTxBody(rawTx.body),
-    rawTx.nativeScriptWitnesses,
-    rawTx.auxiliaryData,
-]
+export const encodeRawTx = (rawTx: RawTransaction) => {
```

Comment:
What's the return value type? It does not seem to be a CBOR as it is claimed in somwhere in the docstring above.

Reply:
I'm willing to accept not naming/describing the return type in detail, it won't get much better with cardano-cli raw tx format being what it is. The confusing part is that the documentation for `encodeRawTx` says
```
Takes a `TransactionBody`, `Transaction` or `RawTransaction` object and encodes it back to CBOR
```
and now we have a function with the same name that actually does not do it. Let's name it differently, say, `preEncodeRawTx` or `prepareRawTxForEncoding`.

Reply:
> and now we have a function with the same name that actually does not do it. Let's name it differently, say, `preEncodeRawTx` or `prepareRawTxForEncoding`.

I agree, but then we should rename all encoders, as well as parsers, so that the consistency between them is preserved. I will do this in `ja/develop`.